zk_toolbox/crates/config/src/ecosystem.rs

```diff
@@ -214,7 +214,7 @@ impl EcosystemConfig {
 /// Result of checking if the ecosystem exists.
 #[derive(Error, Debug)]
 pub enum EcosystemConfigFromFileError {
-    #[error("Ecosystem configuration not found")]
+    #[error("Ecosystem configuration not found (Could not find 'ZkStack.toml' in <path>: Make sure you have created an ecosystem & are in the new folder `cd path/to/ecosystem/name`)")]
```

Comment:
does it make sense to print the actual path? or no?

Reply:
makes sense to me - Updated